apps/web/__tests__/test/feature/reviews.cy.ts

```diff
@@ -48,4 +48,26 @@ describe('Reviews functionality check.', () => {
         // .checkPaginationVisible()
         // .navigateThroughPages()
   });
+
+  it('Edit review item.', () => {
+    homePage.goToCategory();
+    productListPage.goToProduct();
+
+    reviewPage
+        .scrollToReviews()
+        .clickAddReviewButton()
+
+    myAccount.successLogin();
+
+    reviewPage
+        .postReview('Product was great', 'Jane Doe')
+        .clickEditReviewButton()
+        .editReview('Title edited', 'John Doe', 'This is an edited review message.')
+        .checkReviewEditedSuccessfully('John Doe', 'This is an edited review message.')
+        .addReply()
+        .checkReplyAddedSuccessfully()
+        .editReply('John Doe edited', 'Thank you! edited')
+        .checkReplyEditedSuccessfully('John Doe edited', 'Thank you! edited')
+        .removeReview()
+  });
```

Comment:
you don't have to create a new test for this, you can just chain it to the existing one.

Reply:
done